images/bootstrap/Dockerfile

```diff
@@ -14,6 +14,37 @@
 # limitations under the License.
 
 # Includes basic workspace setup, with gcloud and a bootstrap runner
+FROM fedora:36 AS builder
+
+ENV PKG_CONFIG_PATH="/usr/lib/pkgconfig"
+ENV PODMAN_VERSION="v4.0.3"
+
+WORKDIR /app
+
+ENV GIMME_GO_VERSION=1.16.8
+
+# Cache latest stable golang version
+RUN curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme && \
+    chmod +x /bin/gimme && \
+    mkdir -p /etc/setup.mixin.d/ && \
+    echo 'eval $(gimme ${GIMME_GO_VERSION})' > /etc/setup.mixin.d/golang.sh
+RUN cat /etc/setup.mixin.d/golang.sh
+RUN . /etc/setup.mixin.d/golang.sh
```

Comment:
It seems we don't have go in the PATH ?

https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2130/rehearsal-pull-kubernetes-nmstate-unit-test/1529702244696461312